packages/gatsby-theme-docs/README.md

```diff
@@ -65,6 +65,8 @@ The project structure should contain at least the following files and folders:
 
   - `gaTrackingId` (_optional_): this is the Google Analytics tracking ID. For all sites hosted on the `docs.commercetools.com` domain the ID must be: `UA-38285631-3`.
 
+  - `hubspotTrackingCode` (_optional_): this is HubSpot tracking code.
+
     > For test websites the field should not be set.
```

Comment:
This comment was originally belonging to gaTrackingId, but now belongs to the hubspotTrackingCode.    Please either reword it to GA and hubspot or remove it.   

A better solution would be to tell that if the config is not set nothing is tracked in both flags. .